docs/src/components/productX/XHero.tsx

```diff
@@ -0,0 +1,197 @@
+            }}
+          >
+            <IconImage name="product-advanced" /> MUI X
+          </Typography>
+          <Typography variant="h1" sx={{ my: 2, maxWidth: 500 }}>
+            Performant
+            <br />
+            <GradientText>advanced</GradientText>
+            <br /> components
+          </Typography>
+          <Typography color="text.secondary" sx={{ mb: 3, maxWidth: 500 }}>
+            Build complex and data-rich applications using a growing list of advanced components.
+            We&apos;re kicking it off with the most powerful Data Grid on the market.
+          </Typography>
+          <GetStartedButtons
+            installation="npm install @mui/x-data-grid"
+            sx={{ justifyContent: { xs: 'center', md: 'flex-start' } }}
+          />
+        </Box>
+      }
+      rightSx={{
+        p: { md: 2, lg: 3, xl: 4 },
+      }}
+      right={
+        <React.Fragment>
+          <Paper sx={{ mb: { md: 2, lg: 3, xl: 4 } }}>
+            <Box
+              sx={{
+                textAlign: 'center',
+                py: 1,
+                position: 'relative',
+                borderBottom: '1px solid',
+                borderColor: (theme) =>
+                  theme.palette.mode === 'dark' ? 'primaryDark.600' : 'grey.100',
+              }}
+            >
+              <Typography color="primary.main" fontWeight={600}>
+                Trades, October 2020
+              </Typography>
+            </Box>
+            <Box
+              sx={{
+                height: { md: 300, xl: 370 },
+                '& .MuiDataGrid-root': {
+                  '& .MuiCheckbox-root': {
+                    p: 0.5,
+                    '& > svg': {
+                      fontSize: '1.25rem',
+                    },
+                  },
+                  '& .MuiDataGrid-columnHeaderTitleContainer': {
+                    padding: 0,
+                  },
+                  '& .MuiDataGrid-columnSeparator': {
+                    display: 'none',
+                  },
+                  '& .MuiDataGrid-columnHeaderTitle': {
+                    flexGrow: 1,
+                    fontSize: '0.875rem',
+                  },
+                  '& button, & button > svg': {
+                    fontSize: 16,
+                  },
+                  '& .MuiDataGrid-viewport': {
+                    '& .MuiDataGrid-cell': {
+                      fontSize: '0.875rem',
+                      color: (theme) => theme.palette.text.secondary,
+                    },
+                    '& .MuiInputBase-input': {
+                      fontSize: '0.875rem',
+                      px: 0.5,
+                    },
+                  },
+                },
+              }}
+            >
+              <XGrid
+                {...data}
+                disableSelectionOnClick
+                checkboxSelection
+                hideFooter
+                loading={loading}
+                density="compact"
+              />
```

Comment:
The column resizing feature doesn't work. It seems that we hid the drag zone in CSS.